src/dune

```diff
@@ -1,3 +1,5 @@
 (env
  (_
   (flags :standard -warn-error -a+8 -w +a-4-40-41-42-44-48-70)))
+
+(data_only_dirs runtime-bs)
```

Comment:
Maybe we could just drop it altogether?

Reply:
How is commenting it out via `data_only_dirs` different from just removing? Git has history in place in case it needs to be resurrected - it would be trivial to do so... Not sure about how to inquire about the existence of current users. We can probably find out who introduced that part of the codebase and check with them?